src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -137,6 +137,16 @@ class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         noExceptionThrown()
     }
 
+    def "can build image with different targets"() {
```

Comment:
The test should also remove the image afterward it is done.

Reply:
Task "removeImage" to test build file.